Adyen/Helpers/StringHelpers.swift

```diff
@@ -7,27 +7,43 @@
 import Foundation
 
 /// :nodoc:
-public extension String {
+extension String: AdyenCompatible {}
+
+/// :nodoc:
+extension Optional: AdyenCompatible {}
+
+/// :nodoc:
+public extension AdyenScope where Base == String? {
+
+    /// Returns true is optional string not empty.
```

Comment:
`/// Returns true if optional string is null or not empty.`

Reply:
👍